docs/index.md

```diff
@@ -93,88 +95,128 @@ options:
+<!-- TODO: For more examples of using argNorm as a command line tool, click [here](cli_examples.md). -->
+
+## Quick Example 2: argNorm as a Library
+
+Here is a quick demo of using argNorm as a Python library.
+
+### Code
+
+Save this piece of Python code to a file called `argnorm_tutorial.py`
+
+```
+# Import from argNorm
+from argnorm.general import map_to_aro
+from argnorm.drug_categorization import confers_resistance_to, drugs_to_drug_classes
+
+# Creating a list of input genes to be mapped to the ARO
+list_of_input_genes = ['sul1_2_U12338', 'sul1_3_EU855787', 'sul2_1_AF542061']
+
+# The database from which the `list_of_input_genes` was created is the SARG database
+database = 'sarg'
+
+# Looping through `list_of_input` genes and mapping each gene to the ARO
+# Storing each ARO mapping in the `list_of_aros` list
+list_of_aros = []
+for gene in list_of_input_genes:
+    list_of_aros.append(map_to_aro(gene, 'sarg'))
+print(list_of_aros)
+
+# Looping through `list_of_aros` and finding the drugs to which the each ARO confers resistance to
+# Storing each drug in the `list_of_drugs` list
+list_of_drugs = []
+for aro in list_of_aros:
+    list_of_drugs.append(confers_resistance_to(aro))
+print(list_of_drugs)
+
+# Looping through `list_of_drugs` and finding the superclass/drug class of each drug
+# Storing each superclass/drug class in the `list_of_drug_classes` list
+list_of_drug_classes = []
+for drug in list_of_drugs:
+    list_of_drug_classes.append(drugs_to_drug_classes(drug))
+print(list_of_drug_classes)
 ```
 
-#### Raw
-```bash
-argnorm resfinder -i examples/raw/resfinder.resfinder.orfs.tsv -o outputs/raw/resfinder.resfinder.orfs.tsv
-argnorm resfinder -i examples/raw/resfinder.resfinder.reads.tsv -o outputs/raw/resfinder.resfinder.reads.tsv
+### Explanation
+
+To use argNorm as a library, we must first import it in our Python file:
+
+```
+from argnorm.general import map_to_aro
+from argnorm.drug_categorization import confers_resistance_to, drugs_to_drug_classes
 ```
 
-### AMRFinderPlus
-```bash
-argnorm amrfinderplus -i examples/raw/amrfinderplus.ncbi.orfs.tsv -o outputs/raw/amrfinderplus.ncbi.orfs.tsv
+The `general` module contains the function `map_to_aro` which will return the ARO number of a particular antibiotic resistance gene. The `drug_categorization` module contains the functions `confers_resistance_to` and `drugs_to_drug_classes`. The `confers_resistance_to` function returns the drugs to which a gene confers resistance to. The `drugs_to_drug_classes` function returns the drug class to which a specific drug belongs.
+
+The `map_to_aro` function takes two arguments: `gene` and `database`. `gene` is the name of an antibiotic resistance gene. `database` is the database from which `gene` is taken from.
+
+In this example, a list of genes from the SARG database is used, and `map_to_aro` maps each gene in the list to an ARO term. These ARO terms are also stored in the `list_of_aros` list:
```

Comment:
Are these in `sarg`?

Reply:
They're actually in resfinder, I'm just fixing that and changing the `map_to_aro` function to return a pronto term rather than an ARO number as discussed.